taxonomy-editor-frontend/src/pages/go-to-project/index.tsx

```diff
@@ -36,26 +36,32 @@ const GoToProject = ({ clearNavBarLinks }: Props) => {
     let newProjects: ProjectType[] = [];
 
     if (data) {
-      const backendProjects = data.map(
-        ({
-          id,
-          branch_name,
-          taxonomy_name,
-          description,
-          errors_count,
-          status,
-        }) => {
-          return {
-            id, // needed by MaterialTable as key
-            projectName: id,
-            taxonomyName: toTitleCase(taxonomy_name),
-            branchName: branch_name,
-            description: description,
-            errors_count: errors_count,
-            status: status,
-          };
-        }
-      );
+      const backendProjects = data
+        .map(
+          ({
+            id,
+            branch_name,
+            taxonomy_name,
+            description,
+            errors_count,
+            status,
+          }) => {
+            return {
+              id, // needed by MaterialTable as key
+              projectName: id,
+              taxonomyName: toTitleCase(taxonomy_name),
+              branchName: branch_name,
+              description: description,
+              errors_count: errors_count,
+              status: status,
+            };
+          }
+        )
+        .filter(
+          (project) =>
+            project.taxonomyName !== "Languages" &&
+            project.taxonomyName !== "Countries"
+        );
```

Comment:
I don't think this is necessary as we should reset the staging/prod dbs soon enough

Reply:
Yes this is not production software yet !